html/views/common.js

```diff
@@ -9,7 +9,7 @@ function jammer(k, v) {
 	var inJam = isTrue(WS.state["ScoreBoard.InJam"]);
 
 	if (jammerName == null || jammerName == "") {
-		jammerName = leadJammer ? "Lead" : "";
+		if(!ov)	jammerName = leadJammer ? "Lead" : "";
```

Comment:
Please don't use if without braces, it's fragile.

Reply:
I think `jammerName = leadJammer && !ov ? "Lead" : ""; `  would be better.

Reply:
If you're doing that, put parens around the condition for clarity.

Reply:
Did you forget to push? I'm not seeing the change Speedy requested.

Reply:
I missed the comment. Will push it now.